RELEASING.rst

```diff
@@ -1,4 +1,5 @@
 Here are the steps on how to make a new release.
+
 1. Create a ``release-VERSION`` branch from ``upstream/master``.
 2. Update ``CHANGELOG.rst``.
```

Comment:
Wouldn't it be better to just change this to `HISTORY.rst` instead of renaming the file?